Code/GraphMol/FindStereo.cpp

```diff
@@ -501,9 +561,22 @@ void flagRingStereo(ROMol &mol,
         // find the index of the atom on the opposite side of the even-sized
         // ring
         auto oppositeidx = aring[(ai + half_sz) % sz];
+        bool to_atom_opposite_possible = false;
+        auto opposite_atom = mol.getAtomWithIdx(oppositeidx);
+        for (auto bond : mol.atomBonds(opposite_atom)) {
+          auto bidx = bond->getIdx();
+          if (std::find(bring.begin(), bring.end(), bidx) == bring.end() &&
+              (knownBonds[bidx] ||
+               (possibleBonds && possibleBonds->test(bidx)))) {
```

Comment:
It's probably better to do the cheap tests first:
```suggestion
          if ( (knownBonds[bidx] ||
               (possibleBonds && possibleBonds->test(bidx))) &&
               std::find(bring.begin(), bring.end(), bidx) == bring.end()) {
```